build/ci/phase-template.yml

```diff
@@ -40,21 +40,19 @@ phases:
       displayName: Stage build logs
       inputs:
         sourceFolder: $(Build.SourcesDirectory)
-        contents: 
-          ?(msbuild.*|binclash.log|init-tools.log)
+        contents: '?(msbuild.*|binclash.log|init-tools.log)'
         targetFolder: $(Build.ArtifactStagingDirectory)
     - task: CopyFiles@2
-      displayName: Stage more build logs
+      displayName: Stage test output
       inputs:
-        sourceFolder: $(Build.SourcesDirectory)
-        contents: 
-          bin/**/Microsoft.ML.Predictor.Tests/netcoreapp2.1/TestOutput/**/*
+        sourceFolder: $(Build.SourcesDirectory)/bin
+        contents: '**/TestOutput/**/*'
```

Comment:
Can this be part of the above regex expression? That would avoid running the task twice. something like: 
`?(msbuild.*|binclash.log|init-tools.log|bin/**/TestOutput/**/*)`

Reply:
Eh, gave it a try and apparently the vsts engine didn't like it.

Reply:
🤕 we could probably open an issue in the pipeline-tasks repo and for now we can leave the duplication 😄